### DISCUSSION TITLE
Community PRs merged in the last month : Dec-Jan

### DISCUSSION THREAD

**Discussion by predavid** [dotnet/wpf#5599] [General]
We merged 18 community PRs from mid-Dec to mid-Jan. We expect to merge more as we grow our Development team in the next couple of months. 

1)  Remove unused local variable 'selfStyle' and related conditionals in System.Windows.StyleHelper.GetThemeStyle #4010

 2) Replace the Dictionary with HashSet in AppDomainShutdownMonitor #3932
 
3) Don't double import WinFX targets #2976
 
4) Use the new Marshal.IsTypeVisibleFromCom Api #3279
 
5) Special the capacity to improve performance #3086

6) Using readonly struct in WPF Ink module #3908
 
7) Make CustomGuids as readonly #3907
 
8) Adjust project templates to support .NET 6.0 (and default to it) #3976

9)  Added label "ResourceKey" to trace details when resource is not found #3493

10) Remove always-true condition #3009

11) Reduce memory usage by Array.Empty #3085
 
12) Call SHA1.HashData static helper instead of instantiating SHA1CSP #3318
 
13) Fix a wrong property name in a comment #3926
 
14) Set the lock object as readonly #939
 
15) Remove backup file for solution #2965
 
16) Use Array.Empty instead of creating list in ResourceDictionaryDiagnostics #3851
 
17) Fix Grid SetFinalSize method comment #3909
 
18)  Add clearer win32 exception information in SplashScreen CreateWindow #3923

**Comment by ryalanms** [comment]
Thank you @lindexi, @Nirmal4G, @thomasclaudiushuber, @TysonMN, @Youssef1313, @GrabYourPitchforks, @ForNeVeR, and @kant2002 for your contributions to WPF!

**Comment by ryalanms** [comment]
We had to pause to get ARM64 support in, but we will resume community PR reviews and community test passes this week.  Thank you for all the help here.
